src/pancam.py

```diff
@@ -0,0 +1,22 @@
+from follower import Follower
+
+follower = Follower()
+
+def seek():
+    #tag_data = follower.detect()
+    turn = follower.turn()
```

Comment:
What does this exactly hold? It looks like it's just giving a command as turn() doesn't return anything.